.github/workflows/test.yml

```diff
@@ -207,120 +219,167 @@ jobs:
+          path: control-plane/dist/linux/${{ matrix.arch }}
+      - name: extract consul-cni zip
+        env:
+          ZIP_LOCATION: control-plane/dist/cni/linux/${{ matrix.arch }}
+        run: |
+          cd "${ZIP_LOCATION}"
+          unzip -j *.zip
+      - name: extract control-plane zip
+        env:
+          ZIP_LOCATION: control-plane/dist/linux/${{ matrix.arch }}
+        run: |
+          cd "${ZIP_LOCATION}"
+          unzip -j *.zip
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+      - name: Docker Build (Action)
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          context: control-plane
+          platforms: ${{ matrix.arch }}
+          target: release-default
+          tags: docker.io/hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}-pr-${{ github.sha }}
+
+# TODO (agentless): re-enable once tproxy is supported
 #  acceptance-tproxy:
-#    needs: [get-go-version, unit-cli, dev-upload-docker, unit-acceptance-framework, unit-test-helm-templates]
+#    #needs: [get-go-version, unit-cli, dev-upload-docker, unit-acceptance-framework, unit-test-helm-templates]
 #    needs: dev-upload-docker
-#    uses: hashicorp/consul-k8s/.github/workflows/reusable-acceptance.yml@main
+#    uses: ./.github/workflows/reusable-acceptance.yml
 #    with:
 #      name: acceptance-tproxy
 #      directory: acceptance/tests
 #      go-version: ${{ needs.get-go-version.outputs.go-version }}
-#      additional-flags: "-use-kind -kubecontext=kind-dc1 -secondary-kubecontext=kind-dc2 -enable-transparent-proxy"
+#      additional-flags: "-use-kind -kubecontext=kind-dc1 -secondary-kubecontext=kind-dc2 -enable-transparent-proxy -consul-image=hashicorppreview/consul-enterprise:1.14-dev"
 #      gotestsum-version: 1.8.2
 #    secrets:
 #      CONSUL_ENT_LICENSE: ${{ secrets.CONSUL_ENT_LICENSE }}
-#
-#  acceptance:
-#    #needs: [get-go-version, unit-cli, dev-upload-docker, unit-acceptance-framework, unit-test-helm-templates]
+
+  acceptance:
+    needs: [ get-product-version, dev-upload-docker, get-go-version ]
+    uses: ./.github/workflows/reusable-acceptance.yml
+    with:
+      name: acceptance
+      directory: acceptance/tests
+      go-version: ${{ needs.get-go-version.outputs.go-version }}
+      additional-flags: "-use-kind -kubecontext=kind-dc1 -secondary-kubecontext=kind-dc2 -consul-image=hashicorppreview/consul-enterprise:1.14-dev"
+      gotestsum-version: 1.8.2
+      consul-k8s-image: docker.io/hashicorppreview/${{ github.event.repository.name }}-control-plane:${{ needs.get-product-version.outputs.product-version }}-pr-${{ github.sha }}
+    secrets:
+      CONSUL_ENT_LICENSE: ${{ secrets.CONSUL_ENT_LICENSE }}
```

Comment:
In circleci today we also set a directory for debug logs in case of a failure (https://github.com/hashicorp/consul-k8s/blob/5668d4c877b4abd18aa8cc86d1ccfec29e70b55d/.circleci/config.yml#L501-L502). Where would the logs be in GHA?

Reply:
oh okay, good question. I'll take a look